README.md

```diff
@@ -5,7 +5,9 @@ Get it using: `npm install nativescript@next -g`
 
 * Create, build, and run native apps for iOS and Android using JavaScript or TypeScript *
 
-The NativeScript CLI lets you create, build, and deploy NativeScript-based projects on iOS and Android devices.
+The NativeScript CLI lets you create, build, and deploy [NativeScript][7]-based projects on iOS and Android devices.
+
+![NativeScript CLI diagram](ns-cli.png)
```

Comment:
this way the README will be incorrect in `npm` as the `ns-cli.png` will not be available there. Can you please use full URL here.

Reply:
👍